code/modules/paperwork/photocopier.dm

```diff
@@ -323,11 +323,6 @@
 	
 /obj/machinery/photocopier/proc/copy(var/obj/item/paper/copy)
 	var/copy_as_paper = TRUE
-	if(istype(copy, /obj/item/paper/contract/employment))
-		var/obj/item/paper/contract/employment/E = copy
-		var/obj/item/paper/contract/employment/C = new /obj/item/paper/contract/employment (loc, E.target.current)
-		if(C)
-			copy_as_paper = FALSE
 	if(copy_as_paper)
```

Comment:
can remove copy_as_paper as it's only used to exclude employment contracts

Reply:
Will get to it when I'm off work